adage/analyze/management/commands/import_gene_sample_expr.py

```diff
@@ -109,17 +136,21 @@ def import_data_line(line_num, tokens, samples):
 
     gene_name = tokens[0]
     if not gene_name or gene_name.isspace():
-        raise Exception("Input file line #%d: gene systematic_name (column #1)"
+        raise Exception("Input file line #%d: gene name (column #1)"
                         " is blank" % line_num)
 
     try:
-        gene = Gene.objects.get(systematic_name=gene_name)
+        gene = Gene.objects.get(systematic_name=gene_name, organism=organism)
+    except Gene.MultipleObjectsReturned:
+        raise Exception("Input file line #%d: gene name %s (column #1) matches"
+                        " multiple genes in the database" %
+                        (line_num, gene_name))
     except Gene.DoesNotExist:
         # If a gene is not found in database, generate a warning message
         # and skip this line.
         logger.warn(
-            "Input file line #%d: gene systematic_name (column #1) is not "
-            "found in the database: %s", line_num, gene_name)
+            "Input file line #%d: gene name %s (column #1) not found in "
+            "database", line_num, gene_name)
```

Comment:
I checked python doc and seems that both formats of the arguments will work:
https://docs.python.org/2/library/logging.html#logging.Logger.debug
The arguments of `warning()` method is same as those in `debug()` method.
(I think I read an online article some time ago which argues that the format that I am using has better performance, but I couldn't find the link now).

You are definitely right about the method name `warning`. `warn()` is a method for `warnings` library.
https://docs.python.org/2/library/warnings.html#warnings.warn

Two other management commands `import_data.py` and `import_activity.py` are also using `warn()` instead of `warning()`, I will file another PR after this one to fix them.